From 9283b521245847ce27f258c86207cafc2f93faf5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=A9r=C3=A9mie=20Dautheribes?=
 <jeremie.dautheribes@bootlin.com>
Date: Wed, 26 Jun 2024 12:35:43 +0000
Subject: [PATCH 2/2] arm64: dts: ti/k3-am625-beagleplay.dts: add description
 of Nunchuk on i2c3
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

mikrobus needs to be disabled

Upstream-Status: Inappropriate
Signed-off-by: Jérémie Dautheribes <jeremie.dautheribes@bootlin.com>
Signed-off-by: Antonin Godard <antonin.godard@bootlin.com>
---
 arch/arm64/boot/dts/ti/k3-am625-beagleplay.dts | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/ti/k3-am625-beagleplay.dts b/arch/arm64/boot/dts/ti/k3-am625-beagleplay.dts
index 016474d7e344..3903c2191e9f 100644
--- a/arch/arm64/boot/dts/ti/k3-am625-beagleplay.dts
+++ b/arch/arm64/boot/dts/ti/k3-am625-beagleplay.dts
@@ -822,8 +822,15 @@ it66121_out: endpoint {
 &main_i2c3 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&mikrobus_i2c_pins_default>;
-	clock-frequency = <400000>;
+	clock-frequency = <100000>;
 	status = "okay";
+
+	symlink = "play/mikrobus/i2c";
+
+	nunchuk: joystick@52 {
+		compatible = "nintendo,nunchuk";
+		reg = <0x52>;
+	};
 };
 
 &main_spi2 {
-- 
2.47.3

